src/main/java/com/rocketden/main/service/GameManagementService.java

```diff
@@ -91,9 +98,28 @@ public RoomDto startGame(String roomId, StartGameRequest request) {
     // Initialize and add a game object from a room object
     public void createAddGameFromRoom(Room room) {
         Game game = GameMapper.fromRoom(room);
+        game.setProblems(getProblemsFromDifficulty(room.getDifficulty(), 1));
         currentGameMap.put(room.getRoomId(), game);
     }
 
+    // Choose the problem based on problem difficulty settings (empty list for invalid request)
+    private List<Problem> getProblemsFromDifficulty(ProblemDifficulty difficulty, int n) {
+        ProblemSettingsDto request = new ProblemSettingsDto();
+        request.setDifficulty(difficulty);
+        request.setN(n);
+
+        try {
+            List<Problem> problems = new ArrayList<>();
+            for (ProblemDto problemDto : problemService.getRandomProblems(request)) {
+                problems.add(ProblemMapper.toEntity(problemDto));
+            }
+
+            return problems;
+        } catch (ApiException e) {
+            return new ArrayList<>();
+        }
```

Comment:
Personal preference but if an ApiException occurs (like invalid game setting) I feel like we would want to display that to the user rather than returning just an empty list - I'm open to thoughts on this though

Reply:
I was thinking that the empty `ArrayList` could be a signal of this exception, since I wasn't sure how I would pass along this `ApiException` to the frontend in this framework -- though I agree with you, and I'm open to ideas on how to implement this.

Reply:
I haven't tested this so I could be way off, but if you don't catch the ApiException, is it possible it'll just propagate that error to the frontend?

Reply:
Yeah I think that's right -- I'm running into some other issues after making these changes though, I'll try to work with you to solve them in the morning.

Reply:
The error seems to propagate to the frontend, but never reaches the game page, and seems to cause other errors -- we may need to restructure this. There are no "HARD" difficulty problems in the database, so you can see the error propagation and resulting effects in the [this screencast](https://www.loom.com/share/759c0d2b4b4a41afbf515b2e50d65b11). I can look into fixing this, I'd be interested in your feedback as well.

Reply:
Nice catch - in that case I think we can leave that for a future change bug fix (or possibly just leave it as is). I added a card for it just in case though

Reply:
Ok awesome thanks -- I added the Loom screencast to the card as well.